### PR TITLE
New version: CactusNetwork.CactusBlockchainGUI version 2.0.0

### PR DESCRIPTION
**Pull request by TejasMate**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/122503)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

The package manager window is above the installer.

**Comment by stephengillie** [comment]
Manual Validation ended with: 
image

**Comment by stephengillie** [comment]
Hi @TejaMate,

It appears that the installer switches don't result in an unattended install.

File: manifests/c/CactusNetwork/CactusBlockchainGUI/2.0.0/CactusNetwork.CactusBlockchainGUI.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+PackageVersion: 2.0.0
+Platform:
+- Windows.Desktop
+InstallerType: nullsoft
+Scope: user
+InstallModes:
+- silent
+InstallerSwitches:
+  Silent: --silent
+  SilentWithProgress: --silent
```
Comment (Trenly) [inline manifests/c/CactusNetwork/CactusBlockchainGUI/2.0.0/CactusNetwork.CactusBlockchainGUI.installer.yaml:14]:
```suggestion
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by mdanish-kh — APPROVED** [review]

**Pull request merged**